packages/block-template/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@hashintel/block-template",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Block template",
+  "license": "MIT",
+  "author": "HASH",
+  "main": "src/index.ts",
+  "browser": "dist/main.js",
+  "scripts": {
+    "build": "yarn clean; cross-env NODE_ENV=production webpack --mode production && yarn schema",
```

Comment:
Do you think we should add `**/dist/**` to the `.gitignore` for devs creating a block in this repo?

Reply:
Good idea! I added `dist` to `.gitignore` in https://github.com/blockprotocol/blockprotocol/pull/55/commits/37ccbfb0a121726cb769796ad4dc35804295e235